code/game/objects/structures/props/billboard.dm

```diff
@@ -0,0 +1,54 @@
+/obj/structure/prop/billboard
+	name = "billboard"
+	desc = "A large advertisement board, this one is blank"
+	icon = 'icons/obj/props/billboard.dmi'
+	icon_state = "billboard_blank"
+	bound_width = 96
+	bound_height = 64
+
+/obj/structure/prop/billboard/donkburger
+	name = "\improper Donk-n-Go billboard"
+	desc = "A billboard advertising Donk-n-Go, Donk Co's ever-present and ever-unhealthy fast-food venture: GET IN, GET FED, GET GONE!"
+	icon_state = "billboard_donk_n_go"
+
+/obj/structure/prop/billboard/space_cola
+	name = "\improper Space Cola billboard"
+	desc = "A billboard advertising Space Cola: Relax, have a Spoke."
+	icon_state = "billboard_space_cola"
+
+/obj/structure/prop/billboard/NT
+	name = "\improper NanoTrasen billboard"
+	desc = "A billboard advertising NanoTrasen: A Better Tomorrow, Today."
+	icon_state = "billboard_nanotrasen"
+
+/obj/structure/prop/billboard/NT/defaced
+	name = "defaced NanoTrasen billboard"
+	desc = "A billboard advertising Nanotrasen. Someone's sprayed a message onto it: Fuck Corpo Pigs."
+	icon_state = "billboard_fuck_corps"
+
+/obj/structure/prop/billboard/azik
+	name = "\improper Azik Interstellar billboard"
+	desc = "A billboard advertising Azik Interstellar and their newest model: the Autocrat Solar Sailer. Azik Interstellar: Tau Cetian Refinement for Galactic Necessities."
+	icon_state = "billboard_azik"
+	description_fluff = "Azik Interstellar is a subsidiary of Wulf Aeronautics, producing smaller spacecraft for private civillian use."
+
+/obj/structure/prop/billboard/cvr
+	name = "\improper Charlemagne von Rheinland billboard"
+	desc = "A billboard advertising Charlemagne von Rheinland's Germania-class superyacht. Charlemagne von Rheinland: Die Werft der K�nige."
+	icon_state = "billboard_cvr"
+	description_fluff = "The Germania-class superyacht is a type of luxury vessel, built by Wulf Aeronautics and furnished by a subsidiary of Gilthari Exports."
+
+/obj/structure/prop/billboard/twenty_four_seven
+	name = "\improper 24-Seven billboard"
+	desc = "A billboard advertising 24-Seven's new range of limited-edition Slushee flavours. 24-Seven: All Day, Everyday."
+	icon_state = "billboard_twenty_four_seven"
+
+/obj/structure/prop/billboard/starway
+	name = "\improper Starway Transit billboard"
+	desc = "A billboard advertising Starway Transit's direct flight from Kalmar to New York: only 2000 Thalar for an economy class berth. Starway: Your Ticket to the Stars."
```

Comment:
Thaler
Not 100% certain vibes-wise on this one considering the state of interstellar travel, but its not a big deal esp. if its supposed to have existed pre-Incursion.